clients/src/test/java/org/apache/kafka/clients/admin/KafkaAdminClientTest.java

```diff
@@ -5762,6 +5764,347 @@ public void testDeleteStreamsGroupOffsetsFindCoordinatorNonRetriableErrors() thr
         }
     }
 
+    @Test
+    public void testDescribeStreamsGroups() throws Exception {
+        try (AdminClientUnitTestEnv env = new AdminClientUnitTestEnv(mockCluster(1, 0))) {
+            env.kafkaClient().setNodeApiVersions(NodeApiVersions.create());
+
+            // Retriable FindCoordinatorResponse errors should be retried
+            env.kafkaClient().prepareResponse(prepareFindCoordinatorResponse(Errors.COORDINATOR_NOT_AVAILABLE,  Node.noNode()));
+            env.kafkaClient().prepareResponse(prepareFindCoordinatorResponse(Errors.COORDINATOR_LOAD_IN_PROGRESS,  Node.noNode()));
+            env.kafkaClient().prepareResponse(prepareFindCoordinatorResponse(Errors.NONE, env.cluster().controller()));
+
+            StreamsGroupDescribeResponseData data = new StreamsGroupDescribeResponseData();
+
+            // Retriable errors should be retried
+            data.groups().add(new StreamsGroupDescribeResponseData.DescribedGroup()
+                .setGroupId(GROUP_ID)
+                .setErrorCode(Errors.COORDINATOR_LOAD_IN_PROGRESS.code()));
+            env.kafkaClient().prepareResponse(new StreamsGroupDescribeResponse(data));
+
+            /*
```

Comment:
nit: is the standard way we do multi-line comments? It's minor point so I'll leave it to you.

Reply:
Done